storm-client/src/jvm/org/apache/storm/grouping/LoadAwareShuffleGrouping.java

```diff
@@ -50,10 +69,28 @@
     volatile int[] choices;
     private volatile int[] prepareChoices;
     private AtomicInteger current;
+    private Scope currentScope;
+    private NodeInfo sourceNodeInfo;
+    private List<Integer> targetTasks;
+    private AtomicReference<Map<Integer, NodeInfo>> taskToNodePort;
+    private Map<String, Object> conf;
+    private DNSToSwitchMapping dnsToSwitchMapping;
+    private Map<Scope, List<Integer>> localityGroup;
+    private double HIGHER_BOUND;
```

Comment:
This and below line are against style guide and check style will catch it. `higherBound` and `lowerBound`.

Reply:
Thanks. Fixed it